apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -457,6 +461,8 @@ export const submitChatContents = createAsyncThunk(
     }
 
     thunkAPI.dispatch(clearChatMessagePending());
+    // Send a report that user has started the aichat level - a teacher will view that level is now in progress.
```

Comment:
nit: similarly here, maybe add `...the aichat level after successfully receiving a response from the chatbot - ...`?